opensrp-chw/src/androidTest/java/org/smartregister/chw/activity/ba/ANCRegisterTestsBa.java

```diff
@@ -0,0 +1,144 @@
+import org.smartregister.chw.activity.utils.Configs;
+import org.smartregister.chw.activity.utils.Constants;
+import org.smartregister.chw.activity.utils.Order;
+import org.smartregister.chw.activity.utils.OrderedRunner;
+import org.smartregister.chw.activity.utils.Utils;
+
+import static androidx.test.espresso.Espresso.onView;
+import static androidx.test.espresso.action.ViewActions.click;
+import static androidx.test.espresso.action.ViewActions.closeSoftKeyboard;
+import static androidx.test.espresso.action.ViewActions.typeText;
+import static androidx.test.espresso.assertion.ViewAssertions.matches;
+import static androidx.test.espresso.matcher.ViewMatchers.isDisplayed;
+import static androidx.test.espresso.matcher.ViewMatchers.withHint;
+import static androidx.test.espresso.matcher.ViewMatchers.withId;
+
+//import org.junit.Before;
+
+@LargeTest
+//@RunWith(AndroidJUnit4.class)
+@RunWith(OrderedRunner.class)
+public class ANCRegisterTestsBa {
+    @Rule
+    public ActivityTestRule<LoginActivity> mActivityTestRule = new ActivityTestRule<>(LoginActivity.class);
+
+    @Rule
+    public GrantPermissionRule permissionRule = GrantPermissionRule.grant(android.Manifest.permission.CALL_PHONE);
+
+    @Rule
+    public GrantPermissionRule mRuntimePermissionRule = GrantPermissionRule.grant(Manifest.permission.CAMERA);
+
+    @Rule
+    public GrantPermissionRule mRuntimePermissionRule1 = GrantPermissionRule.grant(Manifest.permission.ACCESS_FINE_LOCATION);
+
+    private Utils utils = new Utils();
+
+    public void setUp() throws InterruptedException {
+        utils.logIn(Constants.BoreshaAfyaConfigUtils.ba_username, Constants.BoreshaAfyaConfigUtils.ba_password);
+    }
+
+    @Test
+    @Order(order = 1)
+    public void searchANCRecord() {
+        utils.openDrawer();
+        onView(ViewMatchers.withSubstring(Constants.GenericConfigUtils.anc))
+                .perform(click());
+        onView(withHint("Search name or ID"))
+                .perform(typeText(Configs.TestConfigHelper.aboveFiveFirstNameTwo), closeSoftKeyboard());
+        onView(ViewMatchers.withSubstring(Configs.TestConfigHelper.aboveFiveFirstNameTwo
+                + " " + Configs.TestConfigHelper.aboveFiveSecondNameTwo))
+                .check(matches(isDisplayed()));
+    }
+    @Test
+    @Order(order = 2)
+    public void confirmANCRecordProfile() {
+        utils.openDrawer();
+        onView(ViewMatchers.withSubstring(Constants.GenericConfigUtils.anc))
+                .perform(click());
+        onView(withHint("Search name or ID"))
+                .perform(typeText(Configs.TestConfigHelper.aboveFiveFirstNameTwo), closeSoftKeyboard());
+        onView(ViewMatchers.withSubstring(Configs.TestConfigHelper.aboveFiveFirstNameTwo
+                + " " + Configs.TestConfigHelper.aboveFiveSecondNameTwo))
+                .check(matches(isDisplayed()));
+    }
+
+    @Test
+    @Order(order = 3)
+    public void confirmANCVisitPage() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [JUnit tests should include assert() or fail()](https://app.codacy.com/manual/OpenSRP/opensrp-unicef-wcaro/pullRequest?prid=5426621)